### PR TITLE
Load Airtable data into Google Maps component

### PR DESCRIPTION
**Pull request by dangalan**
We can probably hold off on merging this until Monday when we have a chance to discuss whether or not we not we wanna go with this component or the Mapbox one.

Didn't really do any styling one where the map is on the page - figured we could do that once we decide which component we wanna use.

Screen Shot 2021-11-24 at 3 48 30 PM

### CODE REVIEW

File: components/Map/Map.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+export const Map = ({
+  defaultCenter,
+  markerPositionToLabelMap,
+  ...props
+}: MapProps): JSX.Element => {
+  const { isLoaded, loadError } = useJsApiLoader({
+      googleMapsApiKey: process.env.NEXT_PUBLIC_GOOGLE_MAPS_JS_API_KEY ?? '',
+  })
```
Comment (dangalan) [inline components/Map/Map.tsx:23]: [useJsApiLoader() documentation](https://react-google-maps-api-docs.netlify.app/#usejsapiloader)

File: components/Map/MapMarker.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { Text } from '@chakra-ui/react'
+import { InfoWindow, Marker, MarkerProps } from '@react-google-maps/api'
+import { useState } from 'react'
+
+export const MapMarker = ({position, title}: MarkerProps): JSX.Element => {
+  const marker = new google.maps.Marker()
+  marker.setOptions({
+    anchorPoint: new google.maps.Point(0, -40),
+  })
```
Comment (dangalan) [inline components/Map/MapMarker.tsx:9]: Needed to offset the `InfoWindow` component from the marker.
- [@react-google-maps/api docs](https://react-google-maps-api-docs.netlify.app/#infowindow)
- [Google Maps API docs](https://developers.google.com/maps/documentation/javascript/reference/info-window#InfoWindow.open)

File: components/Map/MapMarker.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        onClick={() => setShowInfoWindow(true)}
+      />
+      {showInfoWindow &&
+        <InfoWindow
+          position={position}
+          anchor={marker}
+          onCloseClick={() => setShowInfoWindow(false)}
+        >
+          <Text>{title}</Text>
+        </InfoWindow>
```
Comment (dangalan) [inline components/Map/MapMarker.tsx:26]: Maybe I missed something in the `react-mapbox-gl` docs but I think the `@react-google-maps/api` gives us a little more flexibility in rendering info on top of map markers.

**Comment by btmoore4** [comment]
Fixed up merge conflicts and moved out the mapbox

**Pull request merged**